vscode-integration/src/action/operation.ts

```diff
@@ -46,3 +46,13 @@ export class LayoutOperation implements Operation {
             && 'elementIds' in action;
     }
 }
+
+export class NavigateAction implements Operation {
+    static readonly KIND = 'navigate';
+    readonly kind = NavigateAction.KIND;
+    constructor(readonly targetTypeId: string) { }
```

Comment:
This definition is missing the `args` property (See also https://github.com/eclipse-glsp/glsp-client/blob/4e7da984e79b162cf4c5ccc3cedf82ced047ddb9/packages/client/src/features/navigation/navigation-action-handler.ts#L53)

Reply:
Hi, I added the args property.